### PR TITLE
Chore merge main c877cef

### PR DESCRIPTION
**Pull request by julien-deramond**
This PR only gathers https://github.com/twbs/bootstrap/commit/c877cefcef18d6a60c5eaec8df469933e64e212a which is the release commit for Bootstrap v5.3.0-alpha2. Only one commit in order to be easily reviewable.

- [x] https://github.com/twbs/bootstrap/commit/c877cefcef18d6a60c5eaec8df469933e64e212a

### CODE REVIEW

File: dist/css/boosted-grid.css [hunk 1]
```diff
@@ -14,224 +14,9 @@
...
-  --bs-link-color-rgb: 151, 178, 233;
-  --bs-link-hover-color-rgb: 186, 203, 241;
-  --bs-code-color: #ffd2f0;
-  --bs-border-color: #666;
-  --bs-border-color-translucent: rgba(255, 255, 255, 0.15);
-}
-
+/* stylelint-disable-line function-disallowed-list */
+/* stylelint-disable-line function-disallowed-list */
+/* stylelint-disable-line function-disallowed-list */
```
Comment (julien-deramond) [inline dist/css/boosted-grid.css:19]: Won't change anything for the release but this triple comment is interesting. Maybe we should avoid this kind of comments for stylelint.

Comment (julien-deramond) [reply dist/css/boosted-grid.css:19]: In the same spirit, `boosted.min.css` contains thinks like `// scss-docs-end collapse-transition\n\n// stylelint-disable function-disallowed-list\n// scss-docs-start aspect-ratios\n` which are useless. Let's see if they can be removed 🤷 in another issue/PR.

File: site/content/docs/5.3/getting-started/download.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ Pull in Boosted's **source files** into nearly any project with some of the most
 Install Boosted in your Node.js powered apps with [the npm package](https://www.npmjs.com/package/boosted):
 
 ```sh
-npm install boosted@v5.3.0-alpha1
+npm install boosted@v5.3.0-alpha2
```
Comment (julien-deramond) [inline site/content/docs/5.3/getting-started/download.md:70]: We missed gathering a commit at some point from Bootstrap, upstream it's using the config version. I'll do it right after this PR.

**Review by louismaximepiton — APPROVED** [review]
LGTM: :rocket:

File: site/content/docs/5.3/getting-started/download.md (hunk 2 above)
Comment (julien-deramond) [reply site/content/docs/5.3/getting-started/download.md:70]: Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1926

**Pull request merged**